glide.yaml

```diff
@@ -1,50 +1,56 @@
 package: github.com/uber/tchannel-go
+excludeDirs:
+ - benchmark
+ - crossdock
+ - examples
+ - relay/relaytest
+ - testutils
+ - thrift/mocks
 import:
 - package: github.com/apache/thrift
   version: ">=0.9.3, <0.11.0"
   subpackages:
   - lib/go/thrift
 - package: github.com/cactus/go-statsd-client
-  version: master
+  version: ^3
   subpackages:
   - statsd
-- package: github.com/davecgh/go-spew
-  version: master
-  subpackages:
-  - spew
-- package: github.com/jessevdk/go-flags
-  version: master
-- package: github.com/pmezard/go-difflib
-  version: master
-  subpackages:
-  - difflib
 - package: github.com/samuel/go-thrift
-  version: master
   subpackages:
   - parser
-- package: github.com/streadway/quantile
-  version: master
-- package: github.com/stretchr/objx
-  version: master
+- package: golang.org/x/net
+  subpackages:
+  - context
+  - ipv4
+  - ipv6
+- package: github.com/uber-go/atomic
+  version: ^1
+  subpackages:
+  - utils
+- package: github.com/opentracing/opentracing-go
+  version: ^1
+testImport:
+- package: github.com/jessevdk/go-flags
+  version: ^1
+  subpackages:
+  - ext
+  - mocktracer
 - package: github.com/stretchr/testify
-  version: master
+  version: ^1
```

Comment:
Are we using any post-1.0 features? Most of our projects end up with `^1.1.4`.

Reply:
Turns out we are fine with ^1.1 (checked out that version and ran tchannel with it)